lib/testing-components/TestRsc.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import {
+  storyblokInit,
+  apiPlugin,
+  getStoryblokApi,
+  StoryblokComponent,
+  SbReactComponentsMap,
+  SbBlokData,
+} from "@storyblok/react/rsc";
+import StoryblokBridgeLoader from "@storyblok/react/bridge-loader";
+
+interface TestProps {
+  bridge?: boolean;
+  accessToken?: string;
+  components?: SbReactComponentsMap;
+  blok: SbBlokData | false;
+}
+
+const TestRsc = ({ bridge, accessToken, components, blok }: TestProps) => {
+  storyblokInit({
+    accessToken,
+    bridge,
+    use: accessToken ? [apiPlugin] : [],
+    components,
+  });
+
+  const storyblokApi = getStoryblokApi();
+  const apiExists = !!(storyblokApi && typeof storyblokApi.get === "function");
+
+  return (
+    <div>
+      <h2>React Testing React Server Component</h2>
+      <StoryblokComponent blok={blok} />
```

Comment:
This test is missing testing `StoryblokStory` (used when you want to set live editing)

Shall we use it instead of `StoryblokComponent`?

Reply:
@alexjoverm we're using `StoryblokStory` on the integration test. Do you think we should add it here?
I'm asking because this Test Component works as a Server Component as it is, and `StoryblokStory` is a component that we will use when enabling the Live Editing experience

Reply:
Ah, I totally missed that. It's definitely more than enough, thanks @fgiuliani :)